bokehjs/test/integration/regressions.ts

```diff
@@ -881,19 +881,32 @@ describe("Bug", () => {
   })
 
   describe("in issue #10498", () => {
-    it("prevents GridBox from rebuilding when rows or cols properties are modified", async () => {
-      const p1 = fig([300, 300])
-      const p2 = fig([300, 300])
-      p1.scatter({x: [0, 1], y: [0, 1], color: "red"})
-      p2.scatter({x: [1, 0], y: [0, 1], color: "green"})
-      const box = new GridBox({
-        children: [[p1, 0, 0], [p2, 0, 1]],
-        cols: ["300px", "300px"],
-        sizing_mode: "fixed",
-      })
-      const {view} = await display(box, [600, 300])
-      box.cols = ["100px", "500px"]
-      await view.ready
+    const params: Array<"cols" | "rows"> = ["cols", "rows"]
```

Comment:
This can be more tersely written as:
```suggestion
    const params = ["cols", "rows"] as const
```

Reply:
Ah, yes!